integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -0,0 +1,184 @@
+# How to set up and use 3rd-party text embeddings for dense vector search in Elasticsearch
+This guide demonstrates how to deploy and use a text embedding model in Elasticsearch. The model will generate vector representations for text, enabling vector similarity (k-nearest neighbours) search.
+
+Please refer to [this guide](./ICD_Elasticsearch_install_and_setup.md) first to create an Elasticsearch instance and set up Kibana if you haven't already.
+
+## Install the eland library
+Run the command below to install the [eland](https://github.com/elastic/eland) library.
+```bash
+$ python -m pip install "eland[pytorch]"
+```
+This library allows us to pull and deploy a 3rd-party text embedding model to our Elasticsearch instance.
+## Create environment variables for ES credentials
+  ```bash
+  export ES_URL=https://<hostname:port>
+  export ES_USER=<username>
+  export ES_PASSWORD=<password>
+  export ES_CACERT=<path-to-your-cert>
+  ```  
```

Comment:
To keep it consistent, in general, it would be better to export the names of the indices, pipelines here as well? They can then be used consistently later on , all through the process

Reply:
+1

Reply:
Good point! I wonder if it'd be better to do this as part of another PR for all the docs we have to keep them consistent.